apps/web/vibes/soul/sections/products-list-section/index.tsx

```diff
@@ -62,103 +60,101 @@ export function ProductsListSection({
-      <div className="@container">
-        <div className="mx-auto max-w-screen-2xl px-4 py-10 @xl:px-6 @xl:py-14 @4xl:px-8 @4xl:py-12">
-          <div>
-            <Stream fallback={<BreadcrumbsSkeleton />} value={streamableBreadcrumbs}>
-              {(breadcrumbs) =>
-                breadcrumbs && breadcrumbs.length > 1 && <Breadcrumbs breadcrumbs={breadcrumbs} />
-              }
-            </Stream>
-            <div className="flex flex-wrap items-center justify-between gap-4 pb-8 pt-6 text-foreground">
-              <h1 className="flex items-center gap-2 font-heading text-3xl font-medium leading-none @lg:text-4xl @2xl:text-5xl">
-                <Suspense
-                  fallback={
-                    <span className="inline-flex h-[1lh] w-[6ch] animate-pulse rounded-lg bg-contrast-100" />
-                  }
-                >
-                  {title}
-                </Suspense>
-                <Suspense
-                  fallback={
-                    <span className="inline-flex h-[1lh] w-[2ch] animate-pulse rounded-lg bg-contrast-100" />
-                  }
-                >
-                  <span className="text-contrast-300">{totalCount}</span>
-                </Suspense>
-              </h1>
-              <div className="flex gap-2">
-                <Stream
-                  fallback={<SortingSkeleton />}
-                  value={Promise.all([
-                    streamableSortLabel,
-                    streamableSortOptions,
-                    streamableSortPlaceholder,
-                  ])}
-                >
-                  {([label, options, placeholder]) => (
-                    <Sorting
-                      defaultValue={sortDefaultValue}
-                      label={label}
-                      options={options}
-                      paramName={sortParamName}
-                      placeholder={placeholder}
-                    />
-                  )}
-                </Stream>
-                <div className="block @3xl:hidden">
-                  <SidePanel.Root>
-                    <SidePanel.Trigger asChild>
-                      <Button size="medium" variant="secondary">
-                        {filterLabel}
-                        <span className="hidden @xl:block">
-                          <Sliders size={20} />
-                        </span>
-                      </Button>
-                    </SidePanel.Trigger>
-                    <SidePanel.Content title={filterLabel}>
-                      <Suspense>
-                        <FiltersPanel
-                          filters={filters}
-                          paginationInfo={paginationInfo}
-                          resetFiltersLabel={resetFiltersLabel}
-                        />
-                      </Suspense>
-                    </SidePanel.Content>
-                  </SidePanel.Root>
-                </div>
+    <div className="group/products-list-section @container">
```

Comment:
Enable "Hide whitespace" to view a better diff that shows the important changes

![CleanShot 2024-12-29 at 14 42 39@2x](https://github.com/user-attachments/assets/50f72105-f328-45b6-814b-d0baa855435b)